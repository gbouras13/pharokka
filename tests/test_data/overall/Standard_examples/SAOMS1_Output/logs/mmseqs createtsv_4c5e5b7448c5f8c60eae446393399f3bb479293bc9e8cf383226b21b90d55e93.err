Command line: mmseqs createtsv ../../databases/CARD SAOMS1_Output/CARD_target_dir/target_seqs SAOMS1_Output/CARD/results_mmseqs SAOMS1_Output/CARD_results.tsv --full-header --threads 8
