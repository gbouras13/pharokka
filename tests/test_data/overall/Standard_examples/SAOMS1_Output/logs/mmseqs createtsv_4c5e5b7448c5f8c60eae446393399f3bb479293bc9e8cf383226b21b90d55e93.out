createtsv ../../databases/CARD SAOMS1_Output/CARD_target_dir/target_seqs SAOMS1_Output/CARD/results_mmseqs SAOMS1_Output/CARD_results.tsv --full-header --threads 8 

MMseqs Version:                 	13.45111
First sequence as representative	false
Target column                   	1
Add full header                 	true
Sequence source                 	0
Database output                 	false
Threads                         	8
Compressed                      	0
Verbosity                       	3

Time for merging to CARD_results.tsv: 0h 0m 0s 4ms
Time for processing: 0h 0m 0s 13ms
